### DISCUSSION TITLE
Public chat

### DISCUSSION THREAD

**Discussion by ben1lam8** [InboxSDK/InboxSDK#793] [Ideas]
Hey guys, do you already have a public chat channel ?

If yes, can we join ?
If no, which service does your team already use ? Slack ? Discord ?

It could be cool to have synchronous ops discussions to quickly fix and handle issues together, like #789, the one that occured yesterday.

**Comment by aleemstreak** [comment]
We do not have a public chat. Given how geographically diverse our
developers and the community are, we don’t think we’re ata scale yet where
synchronous communication would make sense. GitHub issues seems to be
working for now.
